likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/LMFeedCore.kt

```diff
@@ -96,7 +96,7 @@ object LMFeedCore {
             val deviceId = userMeta.deviceId
 
             val lmFeedClient = LMFeedClient.getInstance()
-            val tokens = if (accessToken == null || refreshToken == null) {
+            val tokens = if (accessToken.isNullOrEmpty() || refreshToken.isNullOrEmpty()) {
                 lmFeedClient.getTokens().data ?: Pair("", "")
```

Comment:
should this be `Pair(null, null)

Reply:
There is a check in data layer regarding this.